internal/notification/senders/email.go

```diff
@@ -5,15 +5,35 @@ import (
 
 	"github.com/zitadel/zitadel/internal/notification/channels"
 	"github.com/zitadel/zitadel/internal/notification/channels/fs"
+	"github.com/zitadel/zitadel/internal/notification/channels/instrumenting"
 	"github.com/zitadel/zitadel/internal/notification/channels/log"
 	"github.com/zitadel/zitadel/internal/notification/channels/smtp"
 )
 
-func EmailChannels(ctx context.Context, emailConfig func(ctx context.Context) (*smtp.Config, error), getFileSystemProvider func(ctx context.Context) (*fs.Config, error), getLogProvider func(ctx context.Context) (*log.Config, error)) (chain *Chain, err error) {
+const smtpSpanName = "smtp.NotificationChannel"
+
+func EmailChannels(
+	ctx context.Context,
+	emailConfig func(ctx context.Context) (*smtp.Config, error),
+	getFileSystemProvider func(ctx context.Context) (*fs.Config, error),
+	getLogProvider func(ctx context.Context) (*log.Config, error),
+	successMetricName,
+	failureMetricName string,
+) (chain *Chain, err error) {
 	channels := make([]channels.NotificationChannel, 0, 3)
-	p, err := smtp.InitSMTPChannel(ctx, emailConfig)
+	p, err := smtp.InitChannel(ctx, emailConfig)
+	// TODO: Why is this error not handled?
```

Comment:
@hifabienne @livio-a any idea? It came with commit 7899a0b851a9fb3a056823756a9d6ac2eb88f690

Reply:
i mean it is handled (by ignoring it 😄 )... i think the idea was to be able to configure multiple channels where the message could be sent, but failing on one of them (including initialization) must not return an error

Reply:
Should we log it? At least a debug log?

Reply:
i would add a debug log

Reply:
done